src/dswx_sar/dswx_ni_runconfig.py

```diff
@@ -282,7 +347,92 @@ def read_rtc_polarization(input_h5_list, freq_list):
     return pol_list
 
 
+def read_rtc_resolution(input_h5_list, freq_list):
```

Comment:
```suggestion
def get_rtc_spacing_list(input_h5_list, freq_list):
```
The function name is not clear.